Yank/storage.py

```diff
@@ -0,0 +1,451 @@
+#!/usr/bin/env/python
+
+# =============================================================================
+# MODULE DOCSTRING
+# =============================================================================
+
+"""
+Classes that store arbitrary NetCDF (or other) options and describe how to handle them
+
+"""
+
+# =============================================================================
+# GLOBAL IMPORTS
+# =============================================================================
+
+import abc
+import copy
+import netCDF4 as nc
+
+import numpy as np
+from simtk import openmm, unit
+
+from . import utils
+
+ABC = abc.ABCMeta('ABC', (object,), {})  # compatible with Python 2 *and* 3
+
+# =============================================================================
+# MODULE FUNCTIONS
+# =============================================================================
+
+# =============================================================================
+# CUSTOM EXCEPTIONS
+# =============================================================================
+
+# =============================================================================
+# GENERIC SAVABLE DATA
+# =============================================================================
+
+
+class StorageInterfaceDirVar(object):
+    """
+    Storage Interface Directory/Variable (SIDV) class is a versatile, dynamic class which gives structure to the
```

Comment:
PEP8 suggest 72 characters line width for docstrings, which is nuts, but limiting it to 80-90 characters makes them more readable when viewing diffs.